tests/run-macros/i8514b/Macro_1.scala

```diff
@@ -0,0 +1,16 @@
+import scala.quoted._
+
+class A[+X[_], -Y]
+class P[T]
+class B extends A[P, String]
+
+inline def test(): Unit = ${ testExpr }
+
+def testExpr(using QuoteContext): Expr[Unit] = {
+  import qctx.tasty._
+
+  val t = '[B].unseal.tpe
+  '{
+    println(${Expr(t.baseClasses.map(b => t.baseType(b).toString))})
+  }
```

Comment:
The `.check` file is quite hard to read. Maybe print it this way:

```scala
val baseTypes = t.baseClasses.map(b => t.baseType(b))
val str = Expr(baseTypes.map(_.show).mkString("\n"))
'{ println($str}) }
```

Reply:
Done

Reply:
Perfect